supplementary/alica_tracing/include/tracing/Trace.h

```diff
@@ -56,17 +51,21 @@ class Trace : public alica::IAlicaTrace
     //        {"wait.time", "waited 10 sec for agent 2"}});
     // BE CAREFUL about what you want to log,
     // These are supposed to be micro logs to be carried over the network.
-    void setLog(std::pair<std::string, std::string> logEntry);
+    void setLog(std::pair<std::string, std::string> logEntry) override;
 
     // When operation being traced by this instance fail, call this api
     // Error traces are highlighted in tracing ui.
-    void markError(const std::string& description);
+    void markError(const std::string& description) override;
+
+    // Explicitly set the trace as finished. Any calls to setTag, setLog & markError after this call leaves
```

Comment:
This comment should maybe go in `IAlicaTrace.h`?

Reply:
done